src/cities/cityConfigs.js

```diff
@@ -3,7 +3,7 @@
 import CityConfig from './CityConfig'
 
 const configs = [
-  new CityConfig('neuburgschrobenhausenwohnraum', 'neuburg.wohnen.integreat-app.de', true, 'Neuburg-Schrobenhausen', null, 'https://www.neuburg-schrobenhausen.de/Impressum.n10.html'),
+  new CityConfig('neuburgschrobenhausenwohnraum', 'https://raumfrei.neuburg-schrobenhausen.de/', true, 'Wohnraumbörse Neuburg-Schrobenhausen', null, 'https://www.neuburg-schrobenhausen.de/Impressum.n10.html'),
```

Comment:
this should only be the hostname, so only 'raumfrei.neuburg-schrobenhausen.de', at least we need it that way in the frontend

Reply:
Changed it :)